features/admin.claims.v1/components/edit/local-claim/edit-additional-properties-local-claims.tsx

```diff
@@ -68,25 +68,35 @@ export const EditAdditionalPropertiesLocalClaims:
 
         const allowedScopes: string = useSelector((state: AppState) => state?.auth?.allowedScopes);
         const featureConfig: FeatureConfigInterface = useSelector((state: AppState) => state.config.ui.features);
+        const hasAttributeUpdatePermissions: boolean = useRequiredScopes(
+            featureConfig?.attributeDialects?.scopes?.update
+        );
 
         const isReadOnly: boolean = useMemo(
-            () =>
-                !hasRequiredScopes(
-                    featureConfig?.attributeDialects,
-                    featureConfig?.attributeDialects?.scopes?.update,
-                    allowedScopes
-                ),
+            () => !hasAttributeUpdatePermissions,
```

Comment:
No need to keep useMemo here. Following is sufficient now.
```
const isReadOnly: boolean = !hasAttributeUpdatePermissions;
```

Reply:
Fixed with d241cc141a36f9f0c457a8d09d32d883541df59c